src/sandstorm/run-bundle.c++

```diff
@@ -2147,6 +2149,33 @@ private:
     }
   }
 
+  void fixSandcatsPermissions(const Config& config) {
+    // An older version of the sandcats installer left various sandcats-related files around owned
+    // by root, rather than the sandstorm server user.
+    // var/sandcats should be 0700, with corrected owner/group
+    if (access("../var/sandcats", F_OK) == 0) {
+        setOwnerGroupAndMode(kj::str("../var/sandcats"), 0700, config.uids.uid, config.uids.gid);
+    }
+    // var/sandcats/{register-log,id_rsa{,.pub,private_combined}} should each be 0640, with corrected
+    // owner/group
+    kj::Vector<const char*> files;
```

Comment:
How about:

    kj::StringPtr files[] = { "register-log", ... };

or:

    static const char* const files[] = { "register-log", ... };

The for() loop should work correctly with either of these without changes, though I'd maybe change the `auto& f` to just `auto f` since these are trivially-copyable types.